brave/src/main/java/brave/SpanCustomizer.java

```diff
@@ -0,0 +1,40 @@
+package brave;
+
+import zipkin.Constants;
+import zipkin.TraceKeys;
+
+/**
+ * Simple interface users can customize a span with. For example, this can add custom tags useful
+ * in looking up spans.
+ *
+ * <p>This type is safer to expose directly to users than {@link Span}, as it has no hooks that
+ * can affect the span lifecycle.
+ */
+public abstract class SpanCustomizer {
```

Comment:
Apologies if there are some conventions behind, but going with simple interface seems like more lightweight solution, what do you think?

```
public interface SpanCustomizer {
}
```